shephard-core/src/main/java/com/devsda/platform/shepherdcore/service/documentservice/ExecutionDocumentService.java

```diff
@@ -90,7 +97,7 @@ public boolean updateExecutionDetails(String objectId, String executionID, Strin
             MongoCollection<Document> collection = ExecutionDocumentServiceHelper.getMongoCollection(this.mongoClient, this.shepherdConfiguration.getDataSourceDetails().getDbname(), this.shepherdConfiguration.getDataSourceDetails().getCollectionname());
 
             if (collection != null) {
-                UpdateResult updateResult = collection.updateOne(getSearchFilter(objectId,executionID), getUpdateOperationOnFullExecutionData(updatedInput), new UpdateOptions());
+                UpdateResult updateResult = collection.updateOne(getSearchFilterForExecutionDetails(objectId,executionID), getUpdateOperationOnFullExecutionData(updatedInput), new UpdateOptions());
                 log.debug("updateDocument() :: database: " + this.shepherdConfiguration.getDataSourceDetails().getDbname() + " and collection: " + this.shepherdConfiguration.getDataSourceDetails().getCollectionname()
```

Comment:
Improve logging. This comment applies for all the loggings. Please check other places for reference. And, follow single standard everywhere.